mods/d2k/weapons/missiles.yaml

```diff
@@ -32,147 +33,105 @@ Bazooka:
 		Explosions: tiny_explosion
 		ImpactSounds: EXPLSML1.WAV
 
+^Missile:
+	Inherits: ^Rocket
+	ReloadDelay: 60
+	Range: 5c512
+	MinRange: 1c0
+	Projectile: Missile
+		Blockable: false
+		Shadow: true
+		HorizontalRateOfTurn: 3
+		RangeLimit: 6c614
+		Inaccuracy: 384
+		CruiseAltitude: 1c0
+		MinimumLaunchAngle: 64
+		VerticalRateOfTurn: 10
+		Image: MISSILE2
+		TrailImage: large_trail
+		Speed: 320
+	Warhead@1Dam: SpreadDamage
+		Spread: 256
+		Damage: 480
+		Versus:
+			none: 15
+			wall: 75
+			building: 60
+			wood: 65
+			light: 90
+			heavy: 100
+			invulnerable: 0
+			cy: 30
+			harvester: 50
+	Warhead@3Eff: CreateEffect
+		Explosions: small_explosion
+
+Bazooka:
+	Inherits: ^Rocket
+
 Rocket:
+	Inherits: ^Rocket
 	ReloadDelay: 30
 	Range: 3c512
-	Report: ROCKET1.WAV
 	Projectile: Missile
-		Inaccuracy: 256
-		Image: RPG
-		HorizontalRateOfTurn: 0
-		TrailImage: bazooka_trail2
-		TrailPalette: effect75alpha
-		TrailInterval: 1
 		Speed: 343
 		RangeLimit: 4c204
 	Warhead@1Dam: SpreadDamage
 		Spread: 160
-		Falloff: 100, 50, 25, 0
 		Damage: 250
 		Versus:
 			none: 25
+			wall: 100
 			building: 50
 			wood: 65
+			light: 100
 			heavy: 50
 			invulnerable: 0
 			cy: 20
 			harvester: 50
-		DamageTypes: Prone50Percent, TriggerProne, SmallExplosionDeath
-	Warhead@2Smu: LeaveSmudge
-		SmudgeType: SandCrater, RockCrater
-		InvalidTargets: Vehicle, Structure
 	Warhead@3Eff: CreateEffect
 		Explosions: rocket_explosion
 		ExplosionPalette: effect75alpha
-		ImpactSounds: EXPLSML1.WAV
 
 TowerMissile:
+	Inherits: ^Missile
```

Comment:
This changes `TrailPalette` from undefined to `effect75alpha`.  Probably a good thing (I haven't tested)?